### PR TITLE
Small fixes to SSUs, APCs, air/fire alarms and SMES (#4034, #3257, #4036, #3354)

### LINKED ISSUE #4034: Suit Storage Units don't decontaminate their contents now
UVing the contents only cleans the blood on them.

### PR DESCRIPTION
**Pull request by ghost**
- Fixed #4034; SSUs decontaminating suits is intended as per the description (_[...] also allows the user to decontaminate the contents through a UV-ray purging cycle_)
- Fixed terminal bugs with SMES (#3257, #4036) and added missing busy checks that resulted in duplicated stuff.
- Fixed fire (and air) alarm bug #3354 and added sounds and message feedback to the construction steps.
- APC and computer construction/deconstruction have busy checks now.
- APCs now have to be wired from the correct direction, just like the SMES.

I plan to refactor wall mounted stuff (esp. air alarms) so they can be a bit more modular than the copypasted mess they are right now.

### CODE REVIEW

File: code/game/machinery/alarm.dm [hunk 1]
```diff
@@ -798,7 +790,9 @@
 			else if(wiresexposed && wires.IsAllCut() && iswirecutter(W))
 				buildstage = 1
 				update_icon()
-				new /obj/item/stack/cable_coil(get_turf(src),5)
+				user.visible_message("<span class='attack'>[user] has cut the wiring from \the [src]!</span>", "You have cut the last of the wiring from \the [src].")
+				playsound(get_turf(src), 'sound/items/Wirecutter.ogg', 50, 1)
+				new /obj/item/stack/cable_coil(get_turf(user),5)
```
Comment (ComicIronic) [inline code/game/machinery/alarm.dm]: You should use pooling for this.
`getFromPool(/obj/item/stack/cable_coil, get_turf(user), 5)`
Cable coils are all pooled.

Comment (wwjnc) [inline code/game/machinery/alarm.dm]: Whoops, I thought I replaced that.

File: code/game/machinery/alarm.dm [hunk 2]
```diff
@@ -939,6 +932,8 @@ FIRE ALARM
 
 	if (istype(W, /obj/item/weapon/screwdriver) && buildstage == 2)
```
Comment (ComicIronic) [inline code/game/machinery/alarm.dm]: This should really be using the standards in screwToggle, but it's not entirely a job for a hotfix PR.

Comment (wwjnc) [inline code/game/machinery/alarm.dm]: I am currently refactoring wall-mounted machinery but that's for a new PR.

**Pull request merged**